ledspicer/data/animations/Pulse.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LEDSpicer version="1.0" type="Animation">
+    <actor
+        type="PipeWire"
+        group="Player1"
+        filter="Normal"
+        mode="VuMeter"
+        direction="Forward"
+        off="DarkCyan"
+        low="Blue"
+        mid="Red"
+        high="White"
+        channel="Both"
+    />
+    <actor
+        type="PipeWire"
+        group="Player2"
+        filter="Normal"
+        mode="VuMeter"
+        direction="Forward"
+        off="Red"
+        low="DarkCyan"
+        mid="Blue"
+        high="Green"
+        channel="Both"
```

Comment:
I recommend using the channel Right (or Left depending the layout) for player 2 and channel Left for player 1